godel_robots/abb/godel_irb2400/godel_irb2400_support/launch/irb2400_blending.launch

```diff
@@ -9,6 +9,8 @@
   <arg name="robot_model_plugin" default="abb_irb2400_descartes/AbbIrb2400RobotModel"/>
   <arg name="debug_rviz" default="false"/>
   <arg name="debug_core" default="false"/> <!--Brings up the surface blending service in debug mode-->
+  <arg name="real_pcd" default="false"/>
+  <arg name="pcd_location" default="$(env HOME)/godel_point_cloud_data/surface4.pcd"/> <!-- path to pcd file of part -->
```

Comment:
This way the user gets a nice error message whenever they forget to specify a filepath